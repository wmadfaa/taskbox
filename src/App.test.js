import App from "./App";

test("renders with no problems", () => {
  expect(App).toMatchSnapshot();
});
