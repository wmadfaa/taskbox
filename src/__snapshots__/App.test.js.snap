// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with no problems 1`] = `[Function]`;
